### ISSUE TITLE
[PL][NPC]Tunnel Rat Geomancer

### ISSUE THREAD

**Issue by cubson** [Atlantiss/BugTracker#3853]
Jest tych mobów za mało albo siedzą pod teksturami, bo pamiętam że było ich więcej. W tej chwili są tylko dwa.

http://www.wowhead.com/npc=1174/tunnel-rat-geomancer

**Comment by Philippe777** [comment]
Yeah, the mine says there should be more when you have track humanoids, but....it's mostly troggs inside. :(
